modules/apps/calendar/calendar-test/src/testIntegration/java/com/liferay/calendar/upgrade/v4_2_1/test/UpgradeCalendarBookingTest.java

```diff
@@ -39,7 +47,14 @@
  * @author István András Dézsi
  */
 @RunWith(Arquillian.class)
-public class UpgradeCalendarBookingTest {
+public class UpgradeCalendarBookingTest extends BaseCTUpgradeProcessTestCase {
```

Comment:
@rijema , we are standardizing the name of the upgrade now that we need to extend from `BaseCTUpgradeProcessTestCase`, please talk with @carolmariaabb to know about the standard.

Reply:
@rijema I will send a PR to Brian where I changed the pattern to:

`CalendarBookingCTUpgradeProcessTest.java`

See https://github.com/liferay-forms/liferay-portal/pull/2583.

Let's wait for his approval.